Doc/library/py_compile.rst

```diff
@@ -124,19 +147,6 @@ byte-code cache files in the directory containing the source code.
       This option is useful when the ``.pycs`` are kept up to date by some
       system external to Python like a build system.
 
-
-.. function:: main(args=None)
-
-   Compile several source files.  The files named in *args* (or on the command
-   line, if *args* is ``None``) are compiled and the resulting byte-code is
-   cached in the normal manner.  This function does not search a directory
-   structure to locate source files; it only compiles files named explicitly.
-   If ``'-'`` is the only parameter in args, the list of files is taken from
-   standard input.
-
-   .. versionchanged:: 3.2
-      Added support for ``'-'``.
```

Comment:
This should be preserved (moved to the new doc section)

Reply:
done